Dockerfile

```diff
@@ -11,11 +12,11 @@ RUN chown -R extractor /home/extractor \
 RUN apt-get update && \
     apt-get upgrade -y && \
     apt-get install -y --no-install-recommends \
-        python3.7 \
+        python3.8 \
         python3-pip && \
-    ln -sfn /usr/bin/python3.7 /usr/bin/python && \
-    ln -sfn /usr/bin/python3.7 /usr/bin/python3 && \
-    ln -sfn /usr/bin/python3.7m /usr/bin/python3m && \
+    ln -sfn /usr/bin/python3.8 /usr/bin/python && \
+    ln -sfn /usr/bin/python3.8 /usr/bin/python3 && \
+    ln -sfn /usr/bin/python3.8m /usr/bin/python3m && \
```

Comment:
For Ubuntu 20.04 we should be able to just install `python` and it'll be version 3.8 or later.